packages/projects-docs/pages/learn/getting-started/setting-up-ios.mdx

```diff
@@ -1,21 +1,46 @@
 ---
 title: Setting up iOS
-description: With this guide you'll learn how to get the most out of our iOS integration
+description: With this guide you'll learn how to get started with the iOS app
 ---
 
 # Setting up iOS
 
 ## Getting Started
 
-You can start using CodeSandbox from the web, VS Code, or even iOS - so you can choose the [most convenient platform](/learn/sandboxes/editors) that fits your workflow. Furthermore, you can easily switch between different clients whenever you want and continue the work on your project where you left off.
+Bring your CodeSandbox projects with you wherever you go with just your iPhone or iPad. Simply download the CodeSandbox app from the [App Store](https://apps.apple.com/us/app/codesandbox/id1423330822) and log in using your CodeSandbox account. If you don't have one yet, you can easily sign up using a GitHub, Google, or Apple account.
```

Comment:
Generally avoiding language that assumes difficulty for users. 
```suggestion
Bring your CodeSandbox projects with you wherever you go with just your iPhone or iPad. First, download the CodeSandbox app from the [App Store](https://apps.apple.com/us/app/codesandbox/id1423330822) and log in using your CodeSandbox account. If you don't have one, you can quickly sign up using a GitHub, Google, or Apple account.
```

Reply:
Good shout!